packages/account-v2/src/modules/TradingExperienceFields/TradingExperienceFields.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { FormDropDownField } from '../../components/FormFields';
+import {
+    binaryOptionsTradingExperienceList,
+    binaryOptionsTradingFrequencyList,
+    cfdTradingExperienceList,
+    cfdTradingFrequencyList,
+    forexTradingExperienceList,
+    forexTradingFrequencyList,
+    otherInstrumentsTradingExperienceList,
+    otherInstrumentsTradingFrequencyList,
+} from '../../constants/tradingInformationList';
+import { tradingExperienceValidations } from './validations';
+
+const tradingExperienceConfig = [
```

Comment:
Looks good & clean. 
Can we extract this into a separate file in the same directory to keep this component small?

Reply:
Updated.